test/src/test/java/hudson/model/queue/WideExecutionTest.java

```diff
@@ -87,6 +95,33 @@ public String getDisplayName() {
         }
     }
 
+    @Issue("JENKINS-30084")
+    @Test
+    /*
+     * this is to test that when the assigned executor is not available the flyweighttask is put into the buildable list,
+     * thus the node will be provisioned.
+     * when the flyweight task is not assigned to an offline executors the buildable list is empty.
+     *
+     */
+    public void flyWeightTaskQueue () throws IOException {
```

Comment:
I would like this to go one step further and really make sure that clouds kick in and the `aws-linux` will be provisioned eventually.